cluster_test.go

```diff
@@ -23,45 +23,39 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var cluster Cluster
-
-func GetFakeCluster() {
-	if cluster == nil {
-		cluster, _ = NewMarathonCluster(FAKE_MARATHON_URL)
-	}
-}
-
 func TestUrl(t *testing.T) {
-	GetFakeCluster()
-	assert.Equal(t, cluster.Url(), FAKE_MARATHON_URL)
+	cluster, _ := newCluster(FAKE_MARATHON_URL)
```

Comment:
It might make sense to rename `FAKE_MARATHON_URL` to `fakeMarathonURL` for consistency.